xbmc/music/infoscanner/MusicInfoScanner.cpp

```diff
@@ -919,6 +925,62 @@ int CMusicInfoScanner::RetrieveMusicInfo(const std::string& strDirectory, CFileI
   return numAdded;
 }
 
+void CMusicInfoScanner::CheckBoxSets(VECALBUMS& albums)
+{
+  /* Strategy: Having scanned the tags of the albums to add to the library,
+   * check for disc subtitles. If an album only has one subtitled disc, we don't
+   * consider it a boxset as its most likely a 'live'version of the album or a 'bonus disc'.
+   * Single disc subtitles are still added to the database so they can be displayed by
+   * skins at appropriate times,  If 'boxset' was added to the albumreleasetype then add the
+   * album regardless of any other rules, using 'dummy' as the contents of strDiscSubtitle as
+   * a flag to do so */
```

Comment:
Block comment style is just `/*....*/` now, doen't need the `*` per line

Reply:
Fixed.